inst/rmarkdown/templates/gigascience_article/skeleton/paper-refs.bib

```diff
@@ -0,0 +1,130 @@
+@Book{Anderberg83,
```

Comment:
Not meant to comment on this line, but is the above `oup.pdf` actually used anywhere?

Reply:
Yes, it's used by the (mandatory) supplied class:

rticles/inst/rmarkdown/templates/gigascience_article/skeleton/oup-contemporary.cls:\def\@ouplogo{oup.pdf}